src/components/structured-list/README.md

```diff
@@ -0,0 +1,37 @@
+### SCSS
+
+#### Modifiers
+
+Use these modifiers with `.bx--structured-list` class.
+
+| Selector                        | Description                                |
+|---------------------------------|--------------------------------------------|
+| .bx--structured-list--border    | Applies border around structured-list      |
+| .bx--structured-list--condensed | Applies condensed styles for all body rows |
+| .bx--structured-list-content--nowrap | Applies `white-space: nowrap;` on content |
+
+
+Use these modifiers with `.bx--structured-list-td` class. 
+
+| Selector                             | Description                               |
+|--------------------------------------|-------------------------------------------|
+| .bx--structured-list-content--nowrap | Applies `white-space: nowrap;` on content |
+
+### Javascript                                                                                                                                 
+
+#### Options
+
+| Option                | Default Selector                                      | Description                                      |
+|-----------------------|-------------------------------------------------------|--------------------------------------------------|
+| `selectorInit`        | `[data-structured-list]`                              | The selector to find the StructuredList element. |
+| `selectorStepElement` | `.bx--structured-list-tbody .bx--structured-list-row` | The selector to find the step element.           |
+
+### FAQ
+
+#### Keydown event
+
+Once `StructuredList` instance is initialized, use structured-list--selection.html and users will be able to make row selections with keyboard similar to radio buttons.
+
+- `up` and `down` arrow keys for navigation (focus and select)
+- `tab` and `shift + tab` for navigation (focus only)
+- `enter` and `space` for selecting
```

Comment:
Todo: mention focus issue in Firefox. 

Basically, in Firefox browser: using `up` and `down` arrow keys to navigate a structured-list will set focus on the hidden `<input type=radio>` elements. My guess is that since we set focus on `<label>`s with JS, Firefox browser will immediately put focus on the associated `<input>`.